proto/v1/observations.proto

```diff
@@ -33,6 +33,7 @@ message EntityObservations {
 message VariableObservations {
   string variable = 1;
   repeated EntityObservations observations_by_entity = 2;
+  map<string, datacommons.StatMetadata> all_available_facets = 3;
```

Comment:
I think it's very valuable to be able to get all_available_facets for a stat var for certain entitites, but I wonder if it might be confusing to put it here in this response. Could it be confusing to a user that the facets here do not have to do with the observations in observations_by_entity? And especially if user chooses to filter by a specific facet, but we're still giving them information on all_available_facets? I also notice that we don't populate this when user asks for all_facets, I understand that the same information will be in the `facets` field, but could the inconsistency be confusing

Reply:
Another option is to have a new API that only returns the facet information.

@pradh any thoughts on this too?

Reply:
@Spaceenter A new cache on all facets would be helpful